src/bundle/Resources/views/themes/admin/ui/field_type/edit/relation_base.html.twig

```diff
@@ -12,7 +12,7 @@
                 <th colspan="5">
                     {{ 'ezobjectrelationlist.relations'|trans|desc('Relations') }}:
                     {% if limit != 0 %}
-                        <small>{{ 'ezobjectrelationlist.limit'|trans({'%limit%': limit})|desc('Limit %limit% max') }}</small>
+                        <small>{{ 'ezobjectrelationlist.limit'|trans({'%limit%': limit})|desc('(max. %limit%)') }}</small>
```

Comment:
```suggestion
                        <small>{{ 'ezobjectrelationlist.limit'|trans({'%limit%': limit})|desc('(max %limit%)') }}</small>
```

Reply:
Beg to differ: https://www.proz.com/kudoz/english/transport-transportation-shipping/6235132-%22max-%22-%22min-%22-or-%22max-%22-%22min-%22-in-uk-english.html